sdk/monitor/Microsoft.Azure.Management.Monitor/src/Generated/Operations.cs

```diff
@@ -84,7 +84,7 @@ public async Task<AzureOperationResponse<OperationListResult>> ListWithHttpMessa
             }
             // Construct URL
             var _baseUrl = Client.BaseUri.AbsoluteUri;
-            var _url = new System.Uri(new System.Uri(_baseUrl + (_baseUrl.EndsWith("/") ? "" : "/")), "providers/microsoft.insights/operations").ToString();
```

Comment:
@HarveyLink After updating the generate script, it pulled in changes for multiple other APIs like this one, with changes to the URI. The issue is that these changes are now breaking tests and causing builds to fail. Example test failure in the build (https://dev.azure.com/azure-sdk/public/_build/results?buildId=1169052&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=3374e909-b3a0-5fde-4fba-1f1bb546637f):

 System.Collections.Generic.KeyNotFoundException : Unable to find a matching HTTP request for URL 'GET /providers/Microsoft.Insights/operations?api-version=2015-04-01'. Calling method Item().
                       Stack Trace:
                          at Microsoft.Azure.Test.HttpRecorder.Records.get_Item(String key)
                        at Microsoft.Azure.Test.HttpRecorder.HttpMockServer.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)


Could you please advise on how to fix these tests?